### ISSUE TITLE
Frequency values should be capitalized

### ISSUE THREAD

**Issue by ondrowan** [fmeringdal/rust-rrule#97]
All properties and values in an iCal document should be capitalized. However the `Frequency` doesn't produce a valid value when stringified. E.g.:

```rust
let rrule = RRule::new(Frequency::Yearly)
    .by_month_day((24..=26).collect())
    .by_month(&[Month::December]);

println!("{rrule}");
```

results in an invalid RRULE:

> FREQ=yearly;BYMONTH=12;BYMONTHDAY=24,25,26

**Comment by fmeringdal** [comment]
Thanks for reporting. I will add a fix once time allows, unless someone else picks it up before then

**Comment by fmeringdal** [comment]
Fixed in #99
